lib/src/getDefinitions.ts

```diff
@@ -1,9 +1,10 @@
 import type { ClientFeaturesResponse } from "unleash-client";
 import { removeTrailingSlash } from "./utils";
+import { version, devDependencies } from '../package.json'
 
 const defaultUrl = "http://localhost:4242/api/client/features";
 const defaultToken = "default:development.unleash-insecure-api-token";
-const supportedSpecVersion = "4.2.0";
+const supportedSpecVersion = devDependencies["@unleash/client-specification"];
```

Comment:
is it gonna work at runtime and read correct package.json?

Reply:
Yes. Imports are relative to the file, and reading JSON is fairly standard with TS. `package.json` is always bundled, and both `src` and `dist` are on the same level to import it.